src/main/java/build/buildfarm/instance/shard/RedisShardBackplane.java

```diff
@@ -549,6 +550,10 @@ public void start() throws IOException {
     if (config.getRunFailsafeOperation()) {
       startFailsafeOperationThread();
     }
+
+    // Record client start time
+    client.call(
+        jedis -> jedis.set("uptime/" + clientPublicName, Long.toString(new Date().getTime())));
```

Comment:
I'd consider uptime the entire duration.  start_time might be less confusing and you can calculate uptime from it.

Reply:
Good point, I'll update it.